CHANGELOG.md

```diff
@@ -9,8 +9,17 @@ Please note that the date associated with a release is the date the code
 was committed to the `main` branch. This is not necessarily the date that
 the code was deployed.
 
+
+
 ## [Unreleased]
 
+### Changes Made - 2023-09-22
+
+- Enhanced [Buttons DB Migration Process](https://app.clickup.com/t/8678vk9cj) (#8678vk9cj).
```

Comment:
Can you please make this a little more descriptive. What exactly did you enhance?

And given what we discussed during Tara's PR, can you please remove the link and just have the ClickUp number with the `CU-` in front?

Reply:
Thank you Theresa. I updated the description.